lib/tasks/featured_link.rake

```diff
@@ -0,0 +1,23 @@
+namespace :featured_link do
+  desc "Converts any existing featured_links to action links"
+  task convert_to_action_links: :environment do
+    Coronavirus::SubSection.all.each do |sub_section|
+      next if sub_section.action_link_url.blank?
+
+      content_id = GdsApi.publishing_api.lookup_content_id(base_path: sub_section.action_link_url)
+      content_item = GdsApi.publishing_api.get_content(content_id)
+
+      sub_section.action_link_content = content_item["title"]
```

Comment:
Whilst populating `action_link_summary` from the content item description correct, I think I led you astray when it came to populating the title. It looks like title was being populated from within the content.

For example, the title of the link in the [rules and roadmap accordion](https://www.gov.uk/coronavirus) is "Find out what the rules are", but the title of the [actual page](https://www.gov.uk/guidance/covid-19-coronavirus-restrictions-what-you-can-and-cannot-do?priority-taxon=774cee22-d896-44c1-a611-e3109cce8eae) is "(COVID-19) Coronavirus restrictions: what you can and cannot do".

Reply:
Oh ok, so when we have a `featured_link` in the content (e.g. `[My test link](/testing)`) - the `action_link_content` should be `My test link` and not whatever is in `content_item["title"]`?

Reply:
Exactly this. If I'd realised that earlier, I think I would have agreed with your "update them manually" idea. On the bright-side, hopefully you know more about stubbing content items than you did before!